### PR TITLE
duplicate point on cmd+d

### LINKED ISSUE #1742: Can’t undo when duplicate editing line
When I duplicate the editing line, I could not be able to undo it.

If I try to undo, it seems to blink for a second.

### PR DESCRIPTION
**Pull request by dwelle**
fixes #1742

Somewhat of a cop-out, the underlying problem discussed in the issue still stands, but the solution for that particular bug should actually be more on the note of this PR.

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -266,7 +266,7 @@ class App extends React.Component<any, AppState> {
   }
 
   private syncActionResult = withBatchedUpdates((res: ActionResult) => {
-    if (this.unmounted) {
+    if (this.unmounted || res === false) {
```
Comment (lipis) [inline src/components/App.tsx:269]: `!res`?

Comment (dwelle) [reply src/components/App.tsx:269]: well that's what it was called :), short for `result`

Comment (NMinhNguyen) [reply src/components/App.tsx]: I think the suggestion was to use `!res` instead of `res === false` :sweat:

Comment (dwelle) [reply src/components/App.tsx]: Ha, you're right. Anyway, I went with `=== false` to be explicit.

**Pull request merged**